client/components/thank-you/style.scss

```diff
@@ -56,11 +56,6 @@
 	}
 }
 
-.thank-you__step-cta > a.button:not( .is-primary ) {
-	border-color: var( --color-accent );
-	color: var( --color-accent );
-}
-
```

Comment:
I am removing this piece of code because it is applying a different style from what it is in Figma. Removing it does the work.


<img width="842" alt="Screen Shot 2022-04-27 at 07 38 03" src="https://user-images.githubusercontent.com/5039531/165510446-f66f28f3-27a7-46fe-95bb-98c67227511f.png">

Reply:
Hmmm, in that case I would suggest not to remove it. It seems like  `color-scheme` related.

Reply:
Do you suggest we use the color-scheme related version for both buttons then?

For reference, the Figma layout: oU6nEIeiKSg8CayhEtgR6b-fi-6578%3A51625

Reply:
> Do you suggest we use the color-scheme related version for both buttons then?

what would that look like?

Reply:
If we don't change anything on this style we should have it: 
<img width="665" alt="Screen Shot 2022-04-27 at 08 07 36" src="https://user-images.githubusercontent.com/5039531/165514797-a1c48de7-e22f-4b0a-91a6-2847f80304a7.png">

---

If we apply this style for all buttons (and not only `a`), we will have it:
<img width="665" alt="Screen Shot 2022-04-27 at 08 07 36" src="https://user-images.githubusercontent.com/5039531/165514909-77e6bd0f-d06b-448b-9581-e37a3bc237db.png">

---

If we remove this style completely (as in this PR), we will have it:
![](https://user-images.githubusercontent.com/5039531/165511292-6701dafe-bcbe-4300-b4b2-57e175d45b6d.png)

Reply:
> If we apply this style for all buttons (and not only a), we will have it:

this is what we need to do. Probably the component hasn't accounted for buttons initially.